egs/voxforge/asr1/run.sh

```diff
@@ -7,7 +7,7 @@
 . ./cmd.sh
 
 # general configuration
-backend=chainer
```

Comment:
Same comment
Try to avoid to commit your local changes.
Only `git add` related files.